style/base.css

```diff
@@ -4,6 +4,20 @@
     https://jupyterlab.readthedocs.io/en/stable/developer/css.html
 */
 
+html,
+body,
+html * {
+  font-family: 'Roboto' !important;
```

Comment:
let's not do this, this causes all of jupyter to change fonts.

Reply:
resolved